setup/sql/ilDBTemplate.php

```diff
@@ -13135,7 +13135,7 @@ function setupILIASDatabase()
 'class' => array('text', 'ilhistorytablegui'), 'filename' => array('text', 'Services/History/classes/class.ilHistoryTableGUI.php'), 'comp_prefix' => array('text', ''), 'plugin_path' => array('text', ''), 'cid' => array('text', '3z')));
 
 $ilDB->insert("ctrl_classfile", array(
-'class' => array('text', 'ilobjfileaccesssettingsgui'), 'filename' => array('text', 'Services/WebDAV/classes/class.ilObjFileAccessSettingsGUI.php'), 'comp_prefix' => array('text', ''), 'plugin_path' => array('text', ''), 'cid' => array('text', '40')));
+'class' => array('text', 'ilobjfileaccesssettingsgui'), 'filename' => array('text', 'Modules/File/classes/class.ilObjFileAccessSettingsGUI.php'), 'comp_prefix' => array('text', ''), 'plugin_path' => array('text', ''), 'cid' => array('text', '40')));
```

Comment:
@rsheer this is not needed and will be done when a new release is published. you can revert the file.